core/src/com/unciv/models/UncivSound.kt

```diff
@@ -12,5 +15,14 @@ enum class UncivSound(val value: String) {
     Policy("policy"),
     Paper("paper"),
     Whoosh("whoosh"),
-    Silent("")
+    Silent(""),
+    Custom("");
+    companion object {
+        fun getCustom(name: String?): UncivSound {
+            if (name == null || name.isEmpty()) return Click
```

Comment:
name.isNullOrEmpty() ? 
I mean, I think they're exactly the same, but since there's a prebuilt function for this...